src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -213,7 +212,7 @@ class DeepDiveLogParser extends JavaTokenParsers {
     | "[" ~> cond <~ "]"
     )
 
-  def atom = relationName ~ "(" ~ repsep(expr, ",") ~ ")" ^^ {
+  def atom = relationName ~ "(" ~ repsep(pattern, ",") ~ ")" ^^ {
```

Comment:
This looks good.  `Atom` should now hold terms of `List[Patt]` and `Atom` used for head should be changed to `HeadAtom` or simply `Head` with `List[Expr]`.